src/dsp_tools/dsp_tools.py

```diff
@@ -269,6 +359,7 @@ def call_requested_action(
                     server=args.server,
                     user=args.user,
                     password=args.password,
+                    logger_instance=logger,
                     dump=args.dump
```

Comment:
The way I normally have seen logging, you have a logger instance per source code file (which tells you which file you're in, which is incredibly useful to get information from the logs). Is there a reason to not doing it that way and instead passing the logger instance around?